src/simperium/bucket.js

```diff
@@ -31,6 +31,10 @@ Bucket.prototype.update = function( id, data, options, callback ) {
 	return this.store.update( id, data, this.isIndexing, callback );
 };
 
+Bucket.prototype.hasLocalChanges = function( callback ) {
+	callback( false );
```

Comment:
The pattern for callbacks is that the first argument is reserved for an error if there is one, so the default implementation should be:

```
callback( null, false );
```